examples/fabric/v4/cloudRouterConnectivity/routing-protocol-bgp/README.md

```diff
@@ -12,18 +12,21 @@ At minimum, you must set below variables in `terraform.tfvars` file:
   obtained same way as above
 - `rp_name`- Name of routing Protocol
 - `rp_type`- Type of routing Protocol entity, "BGP"
+- `bgp_enabled_` - Enable or Disable BGP ipv4 and ipv6 sessions
```

Comment:
This should be removed in light of the properties being described below that match the example. `_ipv4` and `_ipv6`.